include/picongpu/fields/background/templates/TWTSfast/TWTSfast.hpp

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2014-2020 Alexander Debus
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
```

Comment:
Please add in this file a comment starting with `/** @file ` and describe what this special implementation of the laser is doing. If we have a better place for this documentation move it there.

Reply:
Done that.